clients/src/main/resources/common/message/IncrementalAlterConfigsResponse.json

```diff
@@ -21,7 +21,7 @@
   "fields": [
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
       "about": "Duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
-    { "name": "responses", "type": "[]AlterConfigsResourceResult", "versions": "0+",
+    { "name": "Resources", "type": "[]AlterConfigsResourceResponse", "versions": "0+",
```

Comment:
Hmm.. Given the name `AlterConfigsResourceResponse`, maybe `Responses` is actually a better name. Perhaps we should be changing `AlterConfigsResponse` instead?

Reply:
Changed to standardize them on `Responses` and `AlterConfigsResourceResponse`